crypto: Bugfix - UTD messages showing unexpected red padlock warning (#3924)
matrix-org/matrix-rust-sdk: crates/matrix-sdk-ui/CHANGELOG.md

- `UtdHookManager` no longer re-reports UTD events as late decryptions.
  ([#3480](https://github.com/matrix-org/matrix-rust-sdk/pull/3480))

> - Unable to decrypt messages wrongly showing a red padlock warning (messages sent in clear).

Reviewer: ```suggestion
- Messages that we were unable to decrypt no longer display a red padlock
  (https://github.com/element-hq/crypto-internal/issues/364)
```
